labelbox/schema/data_row_metadata.py

```diff
@@ -203,6 +229,137 @@ def refresh_ontology(self):
         self._raw_ontology = self._get_ontology()
         self._build_ontology()
 
+    def create_schema(self,
+                      name: str,
+                      kind: DataRowMetadataKind,
+                      options: List[str] = None):
+        """ Create metadata schema
+
+        >>> mdo.create_schema(name, kind, options)
+
+        Args:
+            name (str): Name of metadata schema
+            kind (DataRowMetadataKind): Kind of metadata schema as `DataRowMetadataKind`
+            options (List[str]): List of Enum options
+
+        Returns:
+            Created metadata schema as `DataRowMetadataSchema`
+
+        Raises:
+            KeyError: When provided name is not a valid custom metadata
+        """
+        if not isinstance(kind, DataRowMetadataKind):
+            raise ValueError(f"kind '{kind}' must be a `DataRowMetadataKind`")
+
+        upsert_schema = _UpsertCustomMetadataSchemaInput(name=name,
+                                                         kind=kind.value)
+        if options:
+            if kind != DataRowMetadataKind.enum:
+                raise ValueError(
+                    f"Kind '{kind}' must be an Enum, if Enum options are provided"
+                )
+            upsert_enum_options = [
+                _UpsertCustomMetadataSchemaEnumOptionInput(
+                    name=o, kind=DataRowMetadataKind.option.value)
+                for o in options
+            ]
+            upsert_schema.options = upsert_enum_options
+
+        return self._upsert_schema(upsert_schema)
+
+    def update_schema(self, name: str, new_name: str):
```

Comment:
this needs a return type

Reply:
check all of the return types for these new functions it looks like many are missing them.